### PR TITLE
Force start heketi when pending ops found

### PR DESCRIPTION
**Pull request by raghavendra-talur**

### CODE REVIEW

File: apps/glusterfs/app.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func NewApp(configIo io.Reader) *App {
 	// simply going to refuse to start and provide offline tooling to
 	// repair the situation. In the long term we may gain the ability to
 	// auto-rollback or even try to resume some operations.
-	if HasPendingOperations(app.db) {
+	if HasPendingOperations(app.db) && !app.conf.StartWhenPendingOpsExist {
```
Comment (obnoxxx) [inline apps/glusterfs/app.go]: I suggest we output a big fat warning in the log when
```
HasPendingOperations(app.db) && app.conf.StartWhenPendingOpsExist
```
...

File: apps/glusterfs/app_config.go [hunk 2]
```diff
@@ -27,9 +27,10 @@ type GlusterFSConfig struct {
 	Loglevel   string              `json:"loglevel"`
 
 	// advanced settings
-	BrickMaxSize int `json:"brick_max_size_gb"`
-	BrickMinSize int `json:"brick_min_size_gb"`
-	BrickMaxNum  int `json:"max_bricks_per_volume"`
+	BrickMaxSize             int  `json:"brick_max_size_gb"`
+	BrickMinSize             int  `json:"brick_min_size_gb"`
+	BrickMaxNum              int  `json:"max_bricks_per_volume"`
+	StartWhenPendingOpsExist bool `json:"start_when_pending_ops_exist"`
```
Comment (obnoxxx) [inline apps/glusterfs/app_config.go]: Will it also be possible to set this via an env variable? It should!

**Comment by phlogistonjohn** [comment]
Hmm. I take it you didn't notice pr #1062 ?

**Comment by raghavendra-talur** [comment]
@phlogistonjohn I didn't. I will abandon this.

**Pull request closed without merging**